sdk/edgezones/arm-edgezones/test/public/edgezones_operations_test.spec.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * Code generated by Microsoft (R) AutoRest Code Generator.
+ * Changes may cause incorrect behavior and will be lost if the code is regenerated.
+ */
+
+import {
+  env,
+  Recorder,
+  isPlaybackMode,
+  delay,
+} from "@azure-tools/test-recorder";
+import { createTestCredential } from "@azure-tools/test-credential";
+import { assert, beforeEach, afterEach, it, describe } from "vitest";
+import { createRecorder } from "./utils/recordedClient.js";
+import { EdgeZonesClient } from "../../src/edgeZonesClient.js";
+
+export const testPollingOptions = {
+  updateIntervalInMs: isPlaybackMode() ? 0 : undefined,
+};
+
+describe("extendedZones test", () => {
+  let recorder: Recorder;
+  let subscriptionId: string;
+  let client: EdgeZonesClient;
+  let location: string;
+  let resourceGroup: string;
+  let resourcename: string;
+
+  beforeEach(async (context) => {
+    process.env.SystemRoot = process.env.SystemRoot || "C:\\Windows";
+    recorder = await createRecorder(context);
+    subscriptionId = env.SUBSCRIPTION_ID || '';
+    // This is an example of how the environment variables are used
+    const credential = createTestCredential();
+    client = new EdgeZonesClient(credential, subscriptionId, recorder.configureClientOptions({}));
+    location = "eastus";
+    resourceGroup = "myjstest";
+    resourcename = "losangeles";
+
+  });
+
+  afterEach(async function () {
+    await recorder.stop();
+  });
+
+  it("extendedZones register test", async function () {
+    const res = await client.extendedZones.register(
```

Comment:
feel like this shoud be a lro
https://github.com/Azure/azure-sdk-assets/blob/js/edgezones/arm-edgezones_da64d4dab9/js/sdk/edgezones/arm-edgezones/recordings/node/extendedzones_test/recording_extendedzones_register_test.json#L44

Reply:
tsp link: https://github.com/Azure/azure-rest-api-specs/blob/f69a2f240bc8dff432bfa0d7a4274d9b7760c797/specification/edgezones/EdgeZones.Management/main.tsp#L133-L134

swagger link:  https://github.com/Azure/azure-rest-api-specs/blob/f69a2f240bc8dff432bfa0d7a4274d9b7760c797/specification/edgezones/resource-manager/Microsoft.EdgeZones/preview/2024-04-01-preview/edgezones.json#L168-L212

Reply:
I don't think this is an LRO operation but just confirmed with service team here: https://github.com/Azure/azure-rest-api-specs/pull/28138#discussion_r1671849796.